Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,22 @@
+.fat {
```

Comment:
This is good use of components. The styling alongside gives you something that you can use elsewhere.